pkg/occlient/occlient.go

```diff
@@ -843,12 +815,66 @@ func (c *Client) BootstrapSupervisoredS2I(name string, builderImage string, labe
 	return nil
 }
 
+// AddBootstrapInitContainer adds the bootstrap init container to the deployment config
+// dc is the deployment config to be updated
+// dcName is the name of the deployment config
+func (c *Client) AddBootstrapInitContainer(dc *appsv1.DeploymentConfig, dcName string) {
```

Comment:
Does this function need `Client` receiver?

Reply:
No, I guess, I will remove it. Also should I make it private?